### PR TITLE
Some edits to sections and projects

### PR DESCRIPTION
**Pull request by cybette**
@GregSutcliffe please take a look

### CODE REVIEW

File: bots/README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ You can also add some information about which section and/or project the news re
 - 📦️: ansible `:package:`
 - ⛏️: devtools `:pick:`
 - 🐂: Antsibull tooling `:ox:`
-- AWX: AWX Project
```
Comment (GregSutcliffe) [inline bots/README.md:40]: I'd actually prefer to standardize on text reactions for news, it allows us to be precise, avoid confusion, and support a wider range of projects. @cybette what are your thoughts?

File: bots/newsbot-config.toml [hunk 2]
```diff
@@ -90,15 +92,16 @@ order = 109
...
 title = 'AWX Project'
 description = 'AWX provides a web-based user interface, REST API, and task engine built on top of Ansible.'
 website = 'https://github.com/ansible/awx'
-default_section = 'general'
+default_section = 'projects'
 usual_reporters = [
   '@gundalow:ansible.im',
-  '@therealhaoliu:matrix.org'
+  '@therealhaoliu:matrix.org',
+  '@relrod:matrix.org'
```
Comment (GregSutcliffe) [inline bots/newsbot-config.toml:104]: Should we add Lila Yasin here?

**Review by GregSutcliffe — COMMENTED** [review]
The changes all seem fine, and wouldn't break anything. 2 comments though

File: bots/README.md (hunk 1 above)
Comment (cybette) [reply bots/README.md:40]: I'm in two minds about it, on one hand it's more scalable with text, on the other, you get this generated as headings in the newsletter:

### Antsibull 🐂
### Ansible 📦️
### AWX :AWX:

I usually end up removing the :AWX: if I remember.

Comment (cybette) [reply bots/README.md:40]: unless we use the reaction solely for categorizing, but not include them in the rendered output

Comment (GregSutcliffe) [reply bots/README.md:40]: Right, I was about to suggest we update https://github.com/ansible-community/ansible.im/blob/main/bots/newsbot-project.md as well

Comment (cybette) [reply bots/README.md:40]: yeah we can remove `{{project.emoji}}` for now

File: bots/newsbot-config.toml [hunk 3]
```diff
@@ -114,7 +122,7 @@ usual_reporters = []
 emoji = '⚡️' # :zap:
 name = 'core'
 title = 'Ansible-Core'
-description = "The `ansible-core` package contains the base engine and a small subset of modules and plugins. To see what's planned for the next release, look at the [`ansible-core` roadmaps](https://docs.ansible.com/ansible-core/devel/roadmap/ansible_core_roadmap_index.html)."
+description = "The `ansible-core` package contains the base engine and a small subset of modules and plugins. To see what's planned for the next release, look at the [`ansible-core` roadmaps](https://docs.ansible.com/ansible-core/devel/roadmap/ansible_core_roadmap_index.html). Join the Core discussion in the [Ansible Community Forum](https://forum.ansible.com/tag/core)."
```
Comment (GregSutcliffe) [inline bots/newsbot-config.toml:125]: Whilst I appreciate the change, have we agreed that with Core?

File: bots/README.md (hunk 1 above)
Comment (gundalow) [reply bots/README.md:40]: I've removed `{{project.emoji}}`

Comment (GregSutcliffe) [reply bots/README.md:40]: Thanks @gundalow, can you revert this change to match (and the project config on L95 below)

**Comment by GregSutcliffe** [comment]
Updated and rebased to avoid the conflict. How does it look @cybette?

**Comment by cybette** [comment]
LGTM, thanks @GregSutcliffe and @gundalow !

**Pull request merged**